test/ReactWrapper-spec.js

```diff
@@ -7,7 +7,7 @@ import {
   ReactWrapper,
 } from '../src/';
 import sinon from 'sinon';
-import { REACT013 } from '../src/version';
+import { REACT013, REACT15 } from '../src/version';
```

Comment:
it looks like you brought in `itIf` and `REACT15` but aren't using them anywhere?